src/devices/Blinkt/Blinkt.cs

```diff
@@ -0,0 +1,194 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Threading;
+using System.Drawing;
+
+namespace Iot.Device.Blinkt
+{
+    /// <summary>
+    /// Driver for the Pimoroni Blinkt LED strip for Raspbery Pi
+    /// https://shop.pimoroni.com/products/blinkt
+    ///
+    /// This is a strip of 8 pixels that can be indendently controlled over GPIO.
+    ///
+    /// Setting the color is a 2-step process:
+    /// Set the color using Clear, SetPixel, or SetAll
+    /// Call Show to update the physical pixels
+    /// </summary>
+    public class Blinkt : IDisposable
+    {
+        private const int DAT = 23;
+        private const int CLK = 24;
+        private const int BRIGHTNESS = 7;
+
+        /// <summary>
+        /// The number of pixels in the Blinkt strip
+        /// </summary>
+        public const int NUMBER_OF_PIXELS = 8;
+
+        private readonly Color[] _pixels = new Color[NUMBER_OF_PIXELS];
+        private readonly int _sleepTime = 0;
```

Comment:
Are you sure this can always stay 0? Depending on the CPU speed, additional sleep cycles might be needed.